keyboards/sandwich/keeb68/keymaps/default/keymap.c

```diff
@@ -0,0 +1,70 @@
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _FN
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    QMKBEST = SAFE_RANGE,
+    QMKURL
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [_BASE] = LAYOUT_KEEB68(
+        KC_ESC,   	KC_1,    KC_2,		KC_3,	KC_4,	KC_5,	KC_6,	KC_7,	KC_8,	KC_9,	KC_0,	 KC_MINS,	KC_BSPC,	KC_GRV,
+		KC_TAB,		KC_Q,  	 KC_W,  	KC_E,  	KC_R,  	KC_T,  	KC_Y,  	KC_U,  	KC_I,  	KC_O,  	KC_P,    KC_LBRC,  	KC_BSLS, 	KC_DEL,
+        KC_CAPS, 	KC_A, 	 KC_S,  	KC_D,  	KC_F,  	KC_G,  	KC_H,  	KC_J,  	KC_K,  	KC_L,  	KC_SCLN, KC_QUOT,   KC_ENT,  	KC_PGUP,
+        KC_LSFT, 	KC_Z,  	 KC_X, 		KC_C,  	KC_V,  	KC_B,  	KC_N,  	KC_M, 	KC_COMM,KC_DOT,	KC_SLSH, KC_RSFT, 	KC_UP,   	KC_PGDN,
+        KC_LCTL, 	KC_LGUI, KC_LALT,               	KC_SPC, KC_EQL,	KC_RBRC,KC_RALT,MO(_FN),KC_RCTL, KC_LEFT, 	KC_DOWN, 	KC_RGHT
+		
+    ),
+    [_FN] = LAYOUT_KEEB68(
+        _______, 	KC_F1,	 KC_F2, 	KC_F3, 	KC_F4, 	KC_F5, 	KC_F6, 	KC_F7, 	KC_F8,  KC_F9, 	KC_F10,  KC_F11, 	KC_DEL,		KC_PSCREEN,
+		_______,	_______, BL_INC,	_______,_______,_______,_______,_______,_______,_______,_______,_______,	_______,	KC_PAUSE,
+		_______,	BL_TOGG, BL_DEC,	BL_BRTG,_______,_______,_______,_______,_______,_______,_______,_______,	_______,	KC_HOME,
+		_______,	_______, _______,	_______,_______,_______,_______,_______,_______,_______,_______,_______,	KC_VOLU,	KC_END,
+		_______,	RESET, 	 _______,					_______,KC_F12, _______,_______,MO(_FN),_______,_______,	KC_VOLD,	_______
+		
+    )
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case QMKBEST:
+            if (record->event.pressed) {
+                // when keycode QMKBEST is pressed
+                SEND_STRING("QMK is the best thing ever!");
+            } else {
+                // when keycode QMKBEST is released
+            }
+            break;
+        case QMKURL:
+            if (record->event.pressed) {
+                // when keycode QMKURL is pressed
+                SEND_STRING("https://qmk.fm/\n");
+            } else {
+                // when keycode QMKURL is released
+            }
+            break;
+    }
+    return true;
+}
```

Comment:
```suggestion
```
If these keycodes aren't being used, they can be removed.

Reply:
I had been planning on leaving that in to make it easier for the end user to make their own macros by example, but I suppose it can be removed.